docs/logging/docs/03-01-access-loki.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Access logs
```

Comment:
The title of the doc is `access-loki`, and the heading is about accessing logs - I was thinking that maybe we should change the name to `03-01-access-logs.md` since it's not about accessing Loki as the log management system, but just about checking the logs.

Reply:
you're right